mlx_lm/tuner/utils.py

```diff
@@ -9,7 +9,8 @@
 import mlx.optimizers as opt
 from mlx.utils import tree_flatten, tree_unflatten
 
-from ..models.switch_layers import QuantizedSwitchLinear, SwitchLinear
+from mlx_lm.models.switch_layers import QuantizedSwitchLinear, SwitchLinear
```

Comment:
I would prefer not to switch to absolute imports anywhere. We can consider switching in general but if we do we should be consistent about it and make sure we aren't breaking anything.